pkg/api/constants/constants.go

```diff
@@ -0,0 +1,10 @@
+package constants
```

Comment:
I believe I did that to resolve a circular dependency (but that was a while ago).